cmd/mirroring/main.go

```diff
@@ -237,6 +285,36 @@ func main() {
 			log.Fatalf("failed to delete old checkpoints: %v", err)
 		}
 
+		// Search for identities in the log range
+		if len(ids.Identities) > 0 {
+			entries, err := rekor.GetEntriesByIndexRange(context.Background(), rekorClient, startIndex, endIndex)
+			if err != nil {
+				log.Fatalf("error getting entries by index range: %v", err)
+			}
+			idEntries, err := rekor.MatchedIndices(entries, ids)
+			if err != nil {
+				log.Fatalf("error finding log indices: %v", err)
+			}
+
+			if len(idEntries) > 0 {
+				for _, idEntry := range idEntries {
+					log.Printf("Found subject %s, issuer %s at log index %d, uuid %s\n",
+						idEntry.Subject, idEntry.Issuer, idEntry.Index, idEntry.UUID)
+
+					idFile, err := os.OpenFile(*identitiesFoundFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+					if err != nil {
+						idFile.Close()
```

Comment:
use a `defer idFile.Close()` after opening was successful so you can remove the previous calls.